packages/web3-eth2-beaconchain/tsconfig.json

```diff
@@ -1,21 +1,70 @@
 {
   "compilerOptions": {
-    "declaration": true,
-    "outDir": "lib",
-    "strict": true,
-    "esModuleInterop": true,
-    "module": "commonjs",
-    "lib": ["es6"],
-    "target": "es6",
-    "noImplicitAny": true,
-    "noImplicitThis": true,
-    "strictNullChecks": true,
-    "strictFunctionTypes": true,
-    "noEmit": true,
-    "allowSyntheticDefaultImports": false,
-    "baseUrl": ".",
-    "paths": {
-        "web3-eth2-beaconchain": ["."]
-    }
-  }
-}
\ No newline at end of file
+    /* Visit https://aka.ms/tsconfig.json to read more about this file */
+
+    /* Basic Options */
+    // "incremental": true,                   /* Enable incremental compilation */
+    "target": "esnext",                       /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */
```

Comment:
consider targeting a specific version incase future versions are breaking. i.e es2020

Reply:
Updated to `ES2020`